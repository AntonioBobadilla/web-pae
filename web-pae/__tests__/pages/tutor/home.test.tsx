import Home from '@/pages/tutor/home';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import React from 'react';

describe('Tutor Home Profile', () => {
  it('renders tutor home  page unchanged', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
