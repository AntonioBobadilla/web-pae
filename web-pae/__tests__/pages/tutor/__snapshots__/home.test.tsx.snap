// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tutor Home Profile renders tutor home  page unchanged 1`] = `
<div>
  <div
    class="cards"
  >
    <div
      class="card"
    >
      <div
        class="box"
        href="/tutor/profile"
        role="button"
      >
        <img
          class="image"
          src="/images/profileTutor.jpg"
        />
        <p
          class="description"
        >
           
          Profile
           
        </p>
      </div>
    </div>
    <div
      class="card"
    >
      <div
        class="box"
        href="/tutor/assigned-tutorings"
        role="button"
      >
        <img
          class="image"
          src="/images/history.jpeg"
        />
        <p
          class="description"
        >
           
          Assigned tutorings
           
        </p>
      </div>
    </div>
  </div>
</div>
`;
